magento/pwa-studio
# Adding .gitignore to the files comming from @magento/pwa

## Description

Adding a negative rule into `.npmignore` to accept `.gitignore` when we're using the `npm init @magento/pwa`

## Related Issue
Closes #1931.

## Acceptance 


### Verification Stakeholders


### Specification


### Verification Steps
1. Clone this repo.
2. Go to the venia-concept folder
3. Run `npm pack` (https://docs.npmjs.com/misc/developers#testing-whether-your-npmignore-or-files-config-works)
4. Unpack the `tgz` file and see the `.gitignore` file in there

Please, see the npm documentation about the .npmignore to more details - https://docs.npmjs.com/misc/developers#keeping-files-out-of-your-package

## Screenshots / Screen Captures

![print](https://user-images.githubusercontent.com/21162174/67729919-e0697700-f9d1-11e9-97a0-20e02697e7cb.jpg)

## Checklist
* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
